ironfish-cli/src/commands/faucet.ts

```diff
@@ -33,10 +33,18 @@ export class FaucetCommand extends IronfishCommand {
       this.exit(1)
     }
 
-    this.log(ONE_FISH_IMAGE)
-
     const client = await this.sdk.connectRpc()
 
+    const configResponse = await client.getConfig({
+      name: 'networkId',
+    })
```

Comment:
I don't think we can use `getConfig` for this -- the node's config might not have the value of `networkId` that's used for the network definition

https://iron-fish.slack.com/archives/C02PF56HESK/p1678236428921539

Reply:
@ygao76 added an RPC for this in #3622

Reply:
updated the PR using the new RPC getNetworkInfo